api/v1/routes/blog.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.orm import Session
+from api.db.database import get_db
+from api.v1.models.blog import Blog
+from api.v1.schemas.blog import DeleteBlogResponseSchema
+from api.utils.dependencies import get_current_admin
+import logging
+import uuid
+
+
+blog = APIRouter(prefix="/api/v1/blogs", tags=["Blogs"])
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def validate_uuid(id: str):
+    try:
+        uuid.UUID(id)
+        return True
+    except ValueError:
+        return False
+
+
+@blog.delete("/{id}", response_model=DeleteBlogResponseSchema)
+async def delete_blog(id: str, db: Session = Depends(get_db), user: str = Depends(get_current_admin)):
+    if not validate_uuid(id):
```

Comment:
Isn't `get_current_admin` returning a `User` object, why are you using `str` as the type

Reply:
I have adjusted it